src/react-apps/applications/ux-editor/src/containers/EditContainer.tsx

```diff
@@ -261,7 +261,14 @@ class Edit extends React.Component<IEditContainerProps, IEditContainerState> {
         inEditMode: false,
       },
     }, () => {
-      this.handleSaveChange(this.state.component);
+      console.log(this.state.component, this.props.component);
+      /* tslint:disable:no-var-keyword prefer-const */
+      var {required, ...restState} = this.state.component;
+      var {required, ...restProps} = this.props.component;
+      /* tslint:enable:no-var-keyword prefer-const */
+      if (JSON.stringify(restState) !== JSON.stringify(restProps)) {
```

Comment:
Any reason you have to use JSON.stringify here?

Reply:
if I don't use JSON.stringify the answer will always be true

Reply:
Allright.